misc/python/materialize/cli/ci_coverage_pr_report.py

```diff
@@ -214,22 +226,48 @@ def main() -> None:
             test_case = junit_xml.TestCase("Unit Tests", "Code Coverage")
             test_case.add_error_info(message="No coverage for unit tests available")
             test_cases.append(test_case)
-    report = get_report(coverage)
+    (uncovered_report, unit_test_only_report) = get_report(coverage)
+
+    test_case = junit_xml.TestCase("Uncovered Lines in PR", "Code Coverage")
+    if len(uncovered_report):
+        print("Uncovered Lines in PR")
+        # Buildkite interprets the +++ and --- chars at the start of line, put
+        # in a zero-width space as a workaround.
+        print(
+            uncovered_report.replace("\n+++", "\n\u200B+++").replace(
```

Comment:
Minor: Shall we extract a constant with a meaningful name for `\u200B` to make it easier to read?